composer.json

```diff
@@ -34,6 +34,7 @@
 	},
 	"require-dev": {
 		"leafo/lessphp": "~0.4.0",
+		"leafo/scssphp": "^0.3.0",
```

Comment:
I assume the library is semver compliant and pick 0.3 for php 5.3 and 0.6 for php 5.4+